packages/core/src/tracing/trace.ts

```diff
@@ -215,6 +217,47 @@ export const continueTrace = <V>(
   }
 
   const { sentryTrace, baggage } = options;
+  const client = getClient();
+
+  const clientOptions = client?.getOptions();
+  const strictTraceContinuation = clientOptions?.strictTraceContinuation || false; // default for `strictTraceContinuation` is `false` todo(v10): set default to `true`
+
+  const incomingDsc = baggageHeaderToDynamicSamplingContext(baggage);
+  const baggageOrgId = incomingDsc?.org_id;
+
+  let sdkOrgId: string | undefined;
```

Comment:
m: I think we should use the same logic here as for _injecting_ the org id, which is use the specified one if defined, else extract it from dsn? We may also put this into a reusable utility, I suppose!